Firestore/core/src/firebase/firestore/local/leveldb_key.cc

```diff
@@ -793,6 +793,23 @@ std::string LevelDbDocumentTargetKey::SentinelKey(
   return Key(document_key, kInvalidTargetId);
 }
 
+std::string LevelDbDocumentTargetKey::EncodeSentinel(
+    model::ListenSequenceNumber sequence_number) {
+  std::string encoded;
+  OrderedCode::WriteSignedNumIncreasing(&encoded, sequence_number);
+  return encoded;
+}
+
+model::ListenSequenceNumber LevelDbDocumentTargetKey::DecodeSentinel(
+    const absl::string_view& slice) {
+  model::ListenSequenceNumber decoded;
+  absl::string_view tmp(slice.data(), slice.size());
```

Comment:
If you pass the incoming slice by value you don't need to make a defensive copy here.

Reply:
Done.